src/mscorlib/shared/System/Collections/Generic/KeyValuePair.cs

```diff
@@ -47,10 +47,10 @@ internal static string PairToString(object key, object value)
     // and IReadOnlyDictionary<TKey, TValue>.
     [Serializable]
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    public struct KeyValuePair<TKey, TValue>
+    public readonly struct KeyValuePair<TKey, TValue>
```

Comment:
I'm working on the APICompat checks work but I'm not sure if this a compatible change. @jkotas @stephentoub is this an API compatible change?

Reply:
Marking struct as readonly should be compatible change. The other direction (removing readonly on a struct) is not a compatible change.

cc @VSadov

Reply:
Yes this is a compatible change.